### PR TITLE
2735 - Arabic_translation_issues.xlsx, translation.xlsx, Chinese_tier.xlsx

### LINKED ISSUE #2735: AR - translation issues(top priority)
@minotogna 

Plz update the translation for Arabic.

The corrections can be found here with [FAO ID](https://unfao-my.sharepoint.com/:x:/g/personal/pengcheng_lai_fao_org/ETq_fFvJac1FsFBwdXRqpKkBOzxA1biUt2ZEm3NA0HAjxw?e=NlSCid).

Note the **Location** is where you can find the content;
 **ENG** is how it is shown in English;
**Arabic (currently on the platform)**  is how it is shown in Arabic now;
 **Arabic (correction)** is the right Arabic translation.

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #2735

### CODE REVIEW

File: src/i18n/resources/ar/fra.js [hunk 1]
```diff
@@ -163,6 +164,11 @@ module.exports = {
         'تستند التقديرات إلى التوافق المتكرر لقوائم الجرد الوطنية للغابات حيث يكون أحدثها قد مضي عليها اكثر من خمس سنوات ؛ و / أو تقييمات تغير الاستشعار عن بعد من خلال التحليل متعدد الأوقات لفترة تنتهي قبل أكثر من خمس سنوات ؛ أو مقارنة الخرائط المتوافقة بدون تحليل زمني متعدد.',
       low: 'مصادر البيانات الأخرى ، على سبيل المثال ، تقديرات الخبراء ، أو التقديرات المستندة إلى تقييمات غير متوافقة.',
```
Comment (sorja) [inline src/i18n/resources/ar/fra.js:165]: https://github.com/openforis/fra-platform/issues/2735#issuecomment-1586714486

File: src/i18n/resources/ar/fra.js [hunk 2]
```diff
@@ -181,5 +187,9 @@ module.exports = {
         'تطبيق المعادلات التفاضلية العامة أو على مستوى المنطقة الحيوية أو مجموعة من عوامل التحويل الخاصة بالبلد / المنطقة الحيوية وعوامل توسيع الكتلة الحيوية الافتراضية للهيئة الحكومية الدولية المعنية بتغير المناخ.',
```
Comment (sorja) [inline src/i18n/resources/ar/fra.js]: https://github.com/openforis/fra-platform/issues/2735#issuecomment-1586714486

File: src/i18n/resources/ar.js [hunk 3]
```diff
@@ -21,6 +21,32 @@ module.exports.translation = {
   dataSource,
   print,
 
+  anchors: {
+    fra: {
+      2025: {
+        '1a': 'ا) 1',
```
Comment (minotogna) [inline src/i18n/resources/ar.js:27]: Isn't this handled with metadata ?

Comment (minotogna) [inline src/i18n/resources/ar.js:27]: Isn't this handled with metadata ?

**Review by minotogna — APPROVED** [review]

**Review by sorja — APPROVED** [review]

**Pull request merged**